client/client.go

```diff
@@ -261,8 +261,7 @@ func (c *Client) downloadTimestamp() error {
 	}
 	// unlike root, targets and snapshot, always try and download timestamps
 	// from remote, only using the cache one if we couldn't reach remote.
-	raw, err := c.remote.GetMeta(role, maxSize)
-	var s *data.Signed
+	raw, s, err := c.downloadSigned(role, maxSize, nil)
 	if err != nil || len(raw) == 0 {
```

Comment:
After playing with this I think you're right. Unmarshalling an empty string appears to error every way I've tried it.